ci/github-script/labels.js

```diff
@@ -191,6 +225,9 @@ module.exports = async function ({ github, context, core, dry }) {
         Object.assign(itemLabels, await handlePullRequest({ item, stats }))
       } else {
         stats.issues++
+        if (item.labels.some(({ name }) => name === '4.workflow: auto-close')) {
+          if (await handleAutoClose(item)) return
```

Comment:
Q: what's the early return for here?

Maybe `handleAutoClose` should document when/why it returns true/false?